dynd/include/array_functions.hpp

```diff
@@ -26,18 +26,6 @@
 
 #include "wrapper.hpp"
 
-typedef DyND_PyWrapperObject<dynd::nd::array> DyND_PyArrayObject;
-
-inline int DyND_PyArray_Check(PyObject *obj)
-{
-  return DyND_PyWrapper_Check<dynd::nd::array>(obj);
-}
-
-inline int DyND_PyArray_CheckExact(PyObject *obj)
-{
-  return DyND_PyWrapper_CheckExact<dynd::nd::array>(obj);
-}
-
 namespace pydynd {
```

Comment:
Not entirely. I needed to move them into wrapper.hpp (a more natural place for them anyway) to avoid some circular includes.